src/reducers/index.js

```diff
@@ -218,6 +236,21 @@ function initializeRowHeightsAndOffsets(state) {
   });
 }
 
+/**
+ * Initialize column offsets based on the given column props.
+ *
+ * @param {!Object} state
+ * @private
+ */
+function initializeColumnOffsets(state) {
+  const { scrollableColumns } = columnWidths(state);
+  const columnOffsetIntervalTree = new PrefixIntervalTree(scrollableColumns.map(column => column.width));
```

Comment:
The PrefixIntervalTree seems a bit wasteful since the same offsets are stored in ColumnWidths.
We'll need to eliminate ColumnWidths later on when Column Virtualization is done.